core/src/main/resources/hudson/model/AbstractProject/configure-common.jelly

```diff
@@ -48,19 +48,17 @@ THE SOFTWARE.
 
   <p:config-assignedLabel />
 
-  <f:section title="${%Advanced Project Options}">
-    <f:advanced>
-      <p:config-quietPeriod />
-      <p:config-retryCount />
-      <p:config-blockWhenUpstreamBuilding />
-      <p:config-blockWhenDownstreamBuilding />
-      <st:include page="configure-advanced.jelly" optional="true" />
-      <f:entry title="${%Display Name}" field="displayNameOrNull">
-         <f:textbox checkUrl="'${rootURL}/checkDisplayName?displayName='+encodeURIComponent(this.value)+'&amp;jobName='+encodeURIComponent('${h.jsStringEscape(it.name)}')"/>
-      </f:entry>
-      <f:optionalBlock name="keepDependencies" checked="${it.keepDependencies}" title="${%Keep the build logs of dependencies}" help="/help/tasks/fingerprint/keepDependencies.html"/>
-    </f:advanced>
-  </f:section>
+  <f:advanced>
+    <p:config-quietPeriod />
+    <p:config-retryCount />
+    <p:config-blockWhenUpstreamBuilding />
+    <p:config-blockWhenDownstreamBuilding />
+    <st:include page="configure-advanced.jelly" optional="true" />
+    <f:entry title="${%Display Name}" field="displayNameOrNull">
+       <f:textbox checkUrl="'${rootURL}/checkDisplayName?displayName='+encodeURIComponent(this.value)+'&amp;jobName='+encodeURIComponent('${h.jsStringEscape(it.name)}')"/>
+    </f:entry>
+    <f:optionalBlock name="keepDependencies" checked="${it.keepDependencies}" title="${%Keep the build logs of dependencies}" help="/help/tasks/fingerprint/keepDependencies.html"/>
+  </f:advanced>
```

Comment:
@jglick @recena One of the biggest problems with the tabs is that they consume way too much horizontal space. This section title creates a tab called "Advanced Project Options". @daniel-beck and I commented on this PR about removing this tab via JS to conserve space. He argued it should be removed from the Jelly.

...we could postpone this change, but I think it needs to happen, so may as well happen earlier.